cosmos/noble.json

```diff
@@ -52,6 +56,7 @@
       "coinDenom": "ATOM",
       "coinMinimalDenom": "ibc/EF48E6B1A1A19F47ECAEA62F5670C37C0580E86A9E88498B7E393EB6F49F33C0",
       "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cosmoshub/uatom.png",
```

Comment:
@boojamya IBC tokens do not require you to add an image link.
Could you remove this field?